wildlifelicensing/apps/main/payment_utils.py

```diff
@@ -116,3 +127,76 @@ def get(self, request, *args, **kwargs):
         }
 
         return redirect('{}?{}'.format(url, urlencode(params)))
+
+
```

Comment:
For the two classes below, I'm wondering if it's worth moving them to a separate forms, views module (which would also house those other views we came up with before).